recipes/default.rb

```diff
@@ -67,6 +67,9 @@
   key_length 4096
   expire 3650
   notifies :restart, resources(:service => "docker")
+  # Don't restart the Docker service when running inside kitchen-dokken
+  # This would otherwise break tests!
+  only_if { node['hostname'] != 'dokken' }
```

Comment:
It's restarting docker on the host? Interesting.

But well.. why not. Would just suggest to skip the restart, not the TLS certificate generation.

```
notifies :restart, resources(:service => "docker") if node['hostname'] == 'dokken'
```